EcpBtBot/Conversational_Integration.py

```diff
@@ -1,5 +1,21 @@
-def ReadingMessage():
-    print(u"> ", end=u"")
-    return input()
-def SendingMessage(message):
-    return print(message)
\ No newline at end of file
+from EcpBtBot.MainClass import *
+
+def Receiving_Response(message,userid):
+    #initialisation du sujet
+    if "subject" not in dico_users[userid]:
+        dico_users[userid]["subject"] ="intro"
+
+    subject = dico_users[userid]["subject"] 
```

Comment:
probably safer to do `subject = dico_users[userid].get('subject', None)` and `if subject is None: return`.

Reply:
ok